ui/ui-frontend/projects/vitamui-library/src/sass/_checkbox.scss

```
@@ -1,9 +1,41 @@
-.vitamui-checkbox {
-  width: 46px;
-  height: 46px;
-  padding: 14px;
+/* change .criteria-checkbox-choice into .vitamui-checkbox */
+input[type='checkbox'].vitamui-checkbox {
+  &:not([disabled]) {
+    @extend .clickable;
+  }
+
+  margin: 0;
+  background-color: #000;
+  height: 18px;
+  width: 18px;
+  accent-color: var(--vitamui-primary);
+  border-radius: 3px;
+  border: 1px solid var(--Grey-700);
+}
+
+label.vitamui-checkbox {
+  input[type='checkbox'] {
+    @extend .vitamui-checkbox;
+  }
+  &:has(input:not([disabled])) {
+    @extend .clickable;
+  }
+
+  display: inline-flex;
+  flex-direction: row;
+  align-items: center;
+  flex-wrap: wrap;
+  min-width: 24px;
+  height: 24px;
+  padding: 3px;
+
+  &.wide {
+    min-width: 46px;
+    min-height: 46px;
+    padding: 14px;
```

On ne devrait pas avoir "14px", on devrait avoir 10 ou 15
```suggestion
    @extend .p-3;
```

---

c'est un element propre au composant là - et au wide

---

Les espacements internes aux composants devraient également respecter les espacements définis en token dans le design system. Si ce n'est pas le cas, il faut remonter le problème aux UX.